addon/modifiers/style.js

```diff
@@ -1,49 +1,70 @@
-import makeFunctionalModifier from 'ember-functional-modifiers';
+import Ember from 'ember';
 import { assign } from '@ember/polyfills';
 import { dasherize } from '@ember/string';
 import { assert } from '@ember/debug';
 import { typeOf } from '@ember/utils';
 
-// Store holds information which properties were set
-// for an element (key) in last run. This allows us
-// to clean up properties that were set but aren't
-// present in options hash anymore.
-const STORE = new WeakMap();
+const isObject = o => typeof o === 'object' && Boolean(o);
 
-export default makeFunctionalModifier((element, [optionsHash = {}], namedOptions = {}) => {
-  let options = assign({}, optionsHash, namedOptions);
-  let properties = [];
+const getStylesFromOptions = (positional, named) =>
+  assign({}, ...positional.filter(isObject), named);
```

Comment:
This adds support for passing multiple hashes to styles modifier isn't it? I like it. :+1: Maybe we should document it and add test coverage as well.

Reply:
It does! :)